src/editTask.js

```diff
@@ -0,0 +1,114 @@
+const todoList = document.querySelector('.todo-list');
+const clearBtn = document.querySelector('.clear-btn');
+
+const runComplete = (inputEl) => {
+  const localData = JSON.parse(localStorage.getItem('tasks'));
+  localData.forEach((task) => {
+    if (inputEl.checked !== false) {
+      /* eslint-disable */
```

Comment:
- [ ] Kindly avoid disabling linters manually, it would be better to ensure your code passes the linter checks, this will help your code more clean and professional  👍🏿

Reply:
Ok thank you @prolajumokeoni . I have fix the required changes